src/magic.cpp

```diff
@@ -338,8 +310,8 @@ void spell_type::load( const JsonObject &jo, const std::string & )
     spell_class = trait_id( temp_string );
     optional( jo, was_loaded, "energy_source", temp_string, "NONE" );
     energy_source = energy_source_from_string( temp_string );
-    optional( jo, was_loaded, "damage_type", temp_string, "NONE" );
-    dmg_type = damage_type_from_string( temp_string );
+    optional( jo, was_loaded, "damage_type", temp_string, "none" );
+    dmg_type = io::string_to_enum<damage_type>( temp_string );
```

Comment:
```suggestion
    optional( jo, was_loaded, "damage_type", dmg_type, DT_NONE );
```
I think this should work, although I'm less familiar with the `optional` interface.

Reply:
i'll confirm that it does work, as i've been doing something similar with the relic stuff for the past few days.